webpack/webpack.config.js

```diff
@@ -1,63 +1,26 @@
 import _ from 'lodash';
-import webpack from 'webpack';
-import strategies from './strategies';
-import yargs from 'yargs';
-
-const argv = yargs
-  .alias('p', 'optimize-minimize')
-  .alias('d', 'debug')
-  .argv;
-
-const defaultOptions = {
-  development: argv.debug,
-  docs: false,
-  test: false,
-  optimize: argv.optimizeMinimize
-};
-
-export default (options) => {
-  options = _.merge({}, defaultOptions, options);
-  const environment = options.optimize ? 'production' : 'development';
-
-  const config = {
-    entry: {
-      'react-bootstrap': './src/index.js'
-    },
-
-    output: {
-      path: './dist',
-      filename: '[name].js',
-      library: 'ReactBootstrap',
-      libraryTarget: 'umd'
-    },
-
-    externals: [
-      {
-        'react': {
-          root: 'React',
-          commonjs2: 'react',
-          commonjs: 'react',
-          amd: 'react'
-        }
+import baseConfig, { argv } from './base.config';
+
+export default _.extend({}, baseConfig, {
+  entry: {
+    'react-bootstrap': './src/index.js'
+  },
+
+  output: {
+    path: './dist',
+    filename: argv.optimizeMinimize ? '[name].min.js' : '[name].js',
+    library: 'ReactBootstrap',
+    libraryTarget: 'umd'
+  },
+
+  externals: [
+    {
+      'react': {
+        root: 'React',
+        commonjs2: 'react',
+        commonjs: 'react',
+        amd: 'react'
```

Comment:
I wonder if there's a way to build this up based on the [bower dependencies](https://github.com/react-bootstrap/react-bootstrap/blob/v0.22.3/tools/amd/bower.json#L19-L20). Since we are missing `classnames` at the moment. Though something to consider if we add `classnames` here we will need to update the getting started docs to explain that classnames is needed for those using no dependency manager...